### PR TITLE
Fixing typo in Minimax algorithm ('the' should be 'then').

### PR DESCRIPTION
**Pull request by kartiknagappa**

### CODE REVIEW

File: md/Minimax-Decision.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ __function__ MINIMAX-DECISION(_state_) __returns__ _an action_
 
 ---
 __function__ MAX\-VALUE(_state_) __returns__ _a utility value_  
-&emsp;__if__ TERMINAL\-TEST(_state_) __the return__ UTILITY(_state_)  
+&emsp;__if__ TERMINAL\-TEST(_state_) __then return__ UTILITY(_state_)
```
Comment (kartiknagappa) [inline md/Minimax-Decision.md]: Page 166 of AIMA 3rd ed.

File: md/Minimax-Decision.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ __function__ MINIMAX-DECISION(_state_) __returns__ _an action_
...
+&emsp;__if__ TERMINAL\-TEST(_state_) __then return__ UTILITY(_state_)  
 &emsp;_v_ &larr; &minus;&infin;  
 &emsp;__for each__ _a_ __in__ ACTIONS(_state_) __do__  
 &emsp;&emsp;&emsp;_v_ &larr; MAX(_v_, MIN\-VALUE(RESULT(_state_, _a_)))  
 &emsp;__return__ _v_  
 
 ---
 __function__ MIN\-VALUE(_state_) __returns__ _a utility value_  
-&emsp;__if__ TERMINAL\-TEST(_state_) __the return__ UTILITY(_state_)  
+&emsp;__if__ TERMINAL\-TEST(_state_) __then return__ UTILITY(_state_)
```
Comment (kartiknagappa) [inline md/Minimax-Decision.md]: Page 166 of AIMA 3rd ed.

**Comment by Chipe1** [comment]
There is already a PR (#50)  fixing the issue.

**Comment by kartiknagappa** [comment]
Sounds good. I'll close this one then.

**Pull request closed without merging**